lib/src/components/NxToast/__tests__/NxToast.test.tsx

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+
+import React from 'react';
+import { rtlRender, rtlRenderElement } from '../../../__testutils__/rtlUtils';
+import { fireEvent, screen } from '@testing-library/react';
+
+import userEvent from '@testing-library/user-event';
+
+import NxToast from '../NxToast';
+import { NxToastProps } from '../types';
+import { NxErrorAlert } from '../../NxAlert/NxAlert';
+
+describe('NxToast', function() {
+  const simpleProps: NxToastProps = {
+    onClose: () => {},
+    children: <NxErrorAlert>Toast Message</NxErrorAlert>
+  };
+
+  const quickRender = rtlRender(NxToast, simpleProps),
+      renderEl = rtlRenderElement(NxToast, simpleProps);
+
+  it('renders a div with a class .nx-toast', function() {
+    const component = quickRender();
+    const toast = component.container.children[0];
+
+    expect(toast.nodeName).toEqual('DIV');
+    expect(toast).toHaveClass('nx-toast');
```

Comment:
I actually don't think we need to test either of these in RTL's testing philosophy.  We do still sometimes test the tagName when it's an HTML element that has special semantics that the calling code need to know about.  But for a div it doesn't really matter externally, so it doesn't need to be tested.  I also don't think the `nx-toast` class matters externally.